packages/design-system-scripts/gulp/build.js

```diff
@@ -23,6 +23,7 @@ const getSrcGlob = (src, changedPath) =>
     ? [changedPath]
     : [
         `${src}/**/*.{js,jsx,ts,tsx}`,
+        `!${src}/**/*.stories.{js,jsx,ts,tsx}`,
```

Comment:
When you don't fix the logo module, you also need to update line 114. You might as well also update 72 to match. I was modifying the existing ignores for test and definition files like ``!${src}/**/*{.test,.spec,.d,.stories}.{js,jsx,ts,tsx}`,`

Reply:
Ah, yes, good point, updated.